app/src/components/Navigation/LeftNavbar.svelte

```diff
@@ -0,0 +1,127 @@
+<script lang="ts">
+	import clsx from 'clsx';
+
+	import { page } from '$app/stores';
+
+	import WalletMultiButton from '../Wallet/WalletMultiButton.svelte';
+	import BoxIcon from './NavbarIcons/BoxIcon.svelte';
+	import HomeIcon from './NavbarIcons/HomeIcon.svelte';
+	import SendIcon from './NavbarIcons/SendIcon.svelte';
+	import TrackIcon from './NavbarIcons/TrackIcon.svelte';
+
+	$: currentPage = $page.url.pathname;
+	$: isNavbarOpen = false;
+
+	const navigation = [
```

Comment:
I would fit somewhere "+" for adding shipments, i think after the home icon. Then package icon will be listing shipments for forwarders, but anyone can access listing too.  These names are kinda tricky, doesn't leak any functionality which is behind them.

Reply:
e.g shippers for package have no additional meaning, maybe shipments is more appropriate

Reply:
I've been thinking about it and I have a few ideas.